IFC4x3/Properties/s/Stagger_1dWD_52DDFheUkLfvK_Wga/Documentation.md

```diff
@@ -0,0 +1,3 @@
+Lateral displacement of the contact wire to the opposite sides of the track center.  
+Correspond to the contact wire zig-zag distance from the track axis
+Note: Stagger avoids localized wear of pantogragh cotact strips.
```

Comment:
Is this a property of a mechanical fastener?

Reply:
No, you're right from Member!

Reply:
Is this resolved?

Reply:
Yes. Reuse the one from Alex. de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb